bin/latex_to_unicode.py

```diff
@@ -179,16 +179,12 @@ def latex_to_unicode(s):
     s = s.replace(r"\&", "&")
     s = s.replace("`", "‘")
 
-    # Clean up
-    s = re.sub(r"(?<!\\)[{}]", "", s)  # unescaped curly braces
-    s = s.replace(r"\{", "{")
-    s = s.replace(r"\}", "}")
-
     def repl(s):
-        logging.warning("discarding control sequence {}".format(s.group(0)))
+        logging.warning(f"discarding control sequence '{s.group(0)}' from '{s.string}'")
         return ""
 
-    s = re.sub(r"\\[A-Za-z]+ |\\.", repl, s)
+    # Remove inserted space for remaining codes ("\code {...}" -> "\code{...}")
+    s = re.sub(r"(\\[A-Za-z]+) \{", r"\1{", s)
 
     return s
 
```

Comment:
The change here is that LaTeX commands not handled by the above are left in place instead of removed. We also restore the space that was inserted to ease handling. So `\nonexistentcommand` will be left as-is, as will `\cite{citekey}`. The view is that it cannot be our responsibility to ensure these are all correct.